lbry/extras/daemon/daemon.py

```diff
@@ -4020,11 +4022,24 @@ def jsonrpc_collection_list(self, resolve_claims=0, account_id=None, wallet_id=N
         wallet = self.wallet_manager.get_wallet_or_default(wallet_id)
         if account_id:
             account = wallet.get_account_or_error(account_id)
-            collections = account.get_collections
+            collections = partial(
+                self.ledger.get_collections,
+                wallet=wallet,
+                accounts=[account],
+                resolve=resolve,
+                resolve_claims=resolve_claims)
             collection_count = account.get_collection_count
         else:
-            collections = partial(self.ledger.get_collections, wallet=wallet, accounts=wallet.accounts)
-            collection_count = partial(self.ledger.get_collection_count, wallet=wallet, accounts=wallet.accounts)
+            collections = partial(
+                self.ledger.get_collections,
+                wallet=wallet,
+                accounts=wallet.accounts,
+                resolve=resolve,
+                resolve_claims=resolve_claims)
+            collection_count = partial(
+                self.ledger.get_collection_count,
+                wallet=wallet,
+                accounts=wallet.accounts)
```

Comment:
there seems to be a bit of redundancy here. adding `resolve_claims=resolve_claims` to `partial()` is not necessary because you also pass it to `paginate_rows()` which also passes it to the `get_collections` call.